clib/src/realtime.rs

```diff
@@ -211,6 +224,7 @@ pub unsafe extern "C" fn XSynth_Realtime_SetSoundfonts(
             .send_event(SynthEvent::AllChannels(ChannelEvent::Config(
                 ChannelConfigEvent::SetSoundfonts(sfvec),
             )));
+        count
```

Comment:
Why is this necessary?

Reply:
I added it because @KaleidonKep99 and I were testing stuff with omv2 when it wasn't working